paywall/src/pages/home.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Index from '../components/Index'
+
+export default function PaywallHome({ router }) {
```

Comment:
So... I _still_ think the file should be named `paywalHome.js` so that we use the same name for the default export and the file name...

Reply:
done